randovania/gui/dialog/game_export_dialog.py

```diff
@@ -207,7 +207,7 @@ def output_file_validator(output_file: Path) -> bool:
 
 
 def is_directory_validator(line: QtWidgets.QLineEdit) -> bool:
-    return not line.text() or not Path(line.text()).is_dir()
+    return not line.text() or not Path(line.text()).is_dir() or line.text() == "."
```

Comment:
I'm curious why `Path(".").is_dir()` doesn't work.

Reply:
`Path(".").is_dir()` is a directory. I wan't to refuse it.